app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -96,13 +96,18 @@ class ThreadFragment : Fragment() {
             val threadUid = result.first.uid
             setupUi(threadUid)
             setupAdapter(threadUid)
-            threadAdapter.expandedMap = result.second
+            threadAdapter.isExpandedMap = result.second
+            threadAdapter.isThemeTheSameMap = result.third
```

Comment:
What about creating a data class for the results, to be a bit more specific than a triple ?

Reply:
It's only used once here and all three values doesn't really represent a very well defined object